### PR TITLE
bigger body fonts (fixes #78)

### LINKED ISSUE #78: Increase font-size of docs copy on desktop sizes

### PR DESCRIPTION
**Pull request by ngokevin**
# After

screen shot 2016-05-31 at 11 29 33 am

# Before

screen shot 2016-05-27 at 11 24 54 am

### CODE REVIEW

**Comment by cvan** [comment]
I personally like it, but I worry about the lightness and thinness of the font here lending to poor legibility and contrast.

**Comment by ngokevin** [comment]
Okay. I'll try just making the body font bigger, and play with headings light weight.

**Comment by ngokevin** [comment]
Updated to simply just increasing some font-sizes.

- Including code font size (increased size, decreased padding)
- Including table font size by very slightly
- Add more padding to table header row, removed extraneous bottom border from last row

**Comment by cvan** [comment]
feel free to make the sidebar larger (so there isn't as much wrapping):

image

image

**Comment by cvan** [comment]
image

image

File: themes/aframe/source/css/_settings.styl [hunk 1]
```diff
@@ -9,11 +9,12 @@ $logo-font = $body-font
 $mono-font = $code-font
 
 // Font sizes
-$body-font-size = 1rem
-$body-font-weight = normal
+$body-font-size = 1.1rem
+$body-font-weight = 400
 $body-line-height = 1.5
```
Comment (cvan) [inline themes/aframe/source/css/_settings.styl]: can you change this to `1.6`?

and can you add this to `_common.styl`?
```styl
.copy li + li
    margin-top: .35rem
```

Comment (ngokevin) [inline themes/aframe/source/css/_settings.styl]: k

**Comment by cvan** [comment]
r+ looks great to me

**Comment by ngokevin** [comment]
Thanks!

**Pull request merged**